airbyte-cdk/python/airbyte_cdk/sources/lcc/interpolation/interpolated_string.py

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+from airbyte_cdk.sources.lcc.interpolation.jinja import JinjaInterpolation
+
+
+class InterpolatedString:
```

Comment:
just curious, in all these objects why use the stateful pattern of `InterpolatedObject` where `object.eval()` returns `Object` instead of using more of a stateless factory pattern like `ObjectInterpolator` which does `interpolator.eval(string, object_config)`?